architecture/additional_concepts/nuagesdn.adoc

```diff
@@ -0,0 +1,117 @@
+
+[[overview-of-openshift-workflows]]
+
+== Overview of {product-title} workflows
+
+Integrating VSP with the {product-title} application workflow allows business applications to be quickly turned up and updated by removing the network lag faced by DevOps teams. VSP supports different workflows with {product-title} in order to accommodate scenarios where users can choose ease-of-use or complete control using policy-based automation.
+
+[[developer-workflow]]
+
+=== Developer workflow
+
+This workflow is used in developer environments. 
+
+It is called a Developer Workflow because there is very little input required from the developer in setting up the networking. In this workflow, the nuage-openshift-monitor is responsible for creating the VSP constructs (Zone, Subnets etc.) needed to provide appropriate policies and networking for pods created in an  {product-title} project. When a project gets created, a default zone and default subnet for that Project gets created by nuage-openshift-monitor. Also when the default subnet created for a given project gets depleted, the nuage-openshift-monitor dynamically creates additional subnets.
+
+[NOTE]
+====
+Separate VSP Zone is created for each {product-title} project ensuring isolation amongst the projects.
+====
+
+[[operations-workflow]]
+
+=== Operations workflow
+
+This workflow is used by operations teams doing production roll outs of their applications.
+
+In this workflow, the network and security policies are first configured on the VSD in accordance with the rules set by the organization to deploy applications. Administrative users can potentially create multiple zones and subnets and map them to the same project using labels. While spinning up the pods, the user can use the Nuage Labels to specify what network a pod needs to attach to and what network policies need to be applied to it. This allows for sophisticated deployments where inter- and intra-project traffic can be controlled in a fine-grained manner. For example, inter-project communication is enabled on a project by project basis. This may be used to connect projects to common services that are deployed in a shared project.
+
+[[integration-components]]
+
+== Integration Components
+
+Nuage VSP integrates with OpenShift using two main components:
+
+. nuage-openshift-monitor
+. vsp-openshift plugin
+
+[[nuage-openshift-monitor]]
+
+=== nuage-openshift-monitor
+
+The nuage-openshift-monitor is a service that monitors the {product-title} API server for creation of projects, services, users, user-groups etc.
+
+[NOTE]
+=====
+In case of Highly Available (HA) {product-title} cluster with multiple masters, nuage-openshift-monitor process runs on all the masters independently without any change in functionality.
```

Comment:
No it isn't yet. Each master will run this service under systemd. DaemonSets is a fairly new feature, IMO adding that to the mix here will confuse people than clarify.